engine/src/flutter/shell/platform/android/platform_view_android.h

```diff
@@ -26,7 +26,8 @@ namespace flutter {
 class AndroidSurfaceFactoryImpl : public AndroidSurfaceFactory {
  public:
   AndroidSurfaceFactoryImpl(const std::shared_ptr<AndroidContext>& context,
-                            bool enable_impeller);
+                            bool enable_impeller,
+                            bool lazy_shader_mode);
```

Comment:
no action required: I wanted to get rid of this too but it was already a lift getting the flags down below.